pallets/inflation-rewards/src/lib.rs

```diff
@@ -86,8 +88,12 @@ pub mod pallet {
             // Get the number of chains at this block (tanssi + container chain blocks)
             weight += T::DbWeight::get().reads_writes(1, 1);
             let registered_para_ids = T::ContainerChains::current_container_chains();
-            let number_of_chains: BalanceOf<T> =
-                ((registered_para_ids.len() as u32).saturating_add(1)).into();
+
+            let mut number_of_chains: BalanceOf<T> = (registered_para_ids.len() as u32).into();
+
+            if let Some(_) = T::GetSelfChainBlockAuthor::get_block_author() {
```

Comment:
comment why

Reply:
_fixed_ in f117a242086e1c10409e64f069c79ec10d6b391c.